### PR TITLE
Add contributors, and sort alphabetically

### PR DESCRIPTION
**Pull request by chrisn**
See #335

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/media-source/pull/336.html" title="Last updated on Nov 10, 2023, 3:07 AM UTC (fc5123f)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/media-source/336/85445eb...fc5123f.html" title="Last updated on Nov 10, 2023, 3:07 AM UTC (fc5123f)">Diff</a>

### CODE REVIEW

File: media-source-respec.html [hunk 1]
```diff
@@ -51,28 +51,35 @@
 
       mseDefGroupName: "media-source",
       mseContributors: [
```
Comment (wolenetz) [inline media-source-respec.html:53]: A couple others whom I believe merit acknowledgement are:

- Dale Curtis (@dalecurtis) - he helped by advising, reviewing, and socializing feature work. In particular, he discovered internally at Google a strong use case that helped motivate my proposal and development of the `SourceBuffer.changeType()` method for seamless codec and bytestream format switching.
- Chris Wilson (@cwilso) - helped by assisting setup of the WICG media-source incubation after v1 was REC and before a WG chartered for v2 was available. `SourceBuffer.changeType()` and other early potential v2 feature incubations were made possible with his help and advice.

**Review by wolenetz — CHANGES_REQUESTED** [review]
Thank you for doing this. I have some review comments.

File: media-source-respec.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (chrisn) [reply media-source-respec.html:53]: Ah, I'd missed @saschanaz, and @sideshowbarker is Michael(tm) Smith

Comment (chrisn) [reply media-source-respec.html:53]: Excellent, thanks, I have added both.

File: media-source-respec.html [hunk 2]
```diff
@@ -51,28 +51,35 @@
...
-        "Steven Robertson",
-        "Matt Ward",
-        "David Dorwin",
-        "Kevin Streeter",
+        "Glenn Adams",
+        "Jer Noble",
         "Joe Steele",
+        "John Simmons",
+        "Kevin Streeter",
+        "Marcos Caceres",
```
[the remark this answers is not in the document]
Comment (chrisn) [reply media-source-respec.html]: Oops, now fixed.

**Comment by marcoscaceres** [comment]
As a followup, can we just move this into the spec itself? I'm trying to remove the old JS file because it conflicts with ReSpecs processing. Also, ReSpec can include actual contributions based on git history. 
https://respec.org/docs/#id-gh-contributors

That can more  accurately reflect contribution and doesn't need manual maintenance (except in the rare case of a "special thanks").

**Comment by marcoscaceres** [comment]
Actually, I just went and did it.

**Comment by chrisn** [comment]
Thanks @marcoscaceres, that's much improved.

**Pull request merged**